frontend/src/app/core/services/tab.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+    providedIn: 'root',
+})
+export class TabService {
+    private activeTabKey = 'activeTab';
+
+    getActiveTab(): string | null {
+        return localStorage.getItem(this.activeTabKey);
+    }
+
+    setActiveTab(tabRoute: string): void {
+        localStorage.setItem(this.activeTabKey, tabRoute);
+    }
+}
```

Comment:
Why do we need to save a tab in localStorage?
Cannot we assume active tab based on the current active route?

Reply:
Done